lib/watir/elements/html_element.rb

```diff
@@ -0,0 +1,7 @@
+module Watir
+  class HTMLElement < Element
+
+    alias_method :attribute, :attribute_value
```

Comment:
When we're actually loading the elements we need `#attribute` to use this: https://github.com/watir/watir/blob/master/lib/watir/attribute_helper.rb#L40
This is kind of a hacky workaround in case you don't want to rename the one in attribute helper. Or disagree on aliasing attribute in general. We can always keep the alias in PO gem. But matching `Watir::Element#attribute` to `Selenium::WebDriver::Element#attribute` makes sense to me.